docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -254,6 +254,14 @@ Here's an example update when using the `DotNetCoreCLI` task in Azure DevOps:
 +    arguments: '--configuration Release -p:TestingPlatformCommandLineArguments="--report-trx --results-directory $(Agent.TempDirectory) --coverage"'
 ```
 
+## Known limitations
+
+The MSTest SDK has limited integration with NuGet, meaning that the usual NuGet update and Visual Studio UI for managing NuGet packages **DOES NOT** work as expected. See this issue for more details: [NuGet#13127](https://github.com/NuGet/Home/issues/13127).
```

Comment:
If someone is here viewing this section, which is **Known limitations** it's not really beneficial to try and use CAPS to emphasize something. Someone taking the time to read this paragraph will understand what's being said here. It's not hidden in a sea of paragraphs and tables, it's the first thing mentioned. If anything, I would use this instead:

```suggestion
The MSTest SDK has limited integration with NuGet, meaning that the usual NuGet update and Visual Studio UI for managing NuGet packages **doesn't** work as expected. See this issue for more details: [NuGet#13127](https://github.com/NuGet/Home/issues/13127).
```

But honestly this isn't very clear about the limitation. I don't want to have to go click on a link and read through comment-after-comment to understand the limitation. Just tell me the limitation and what I need to do to work around it.

Reply:
@adegeo Could you please suggest a wording. Basically, NuGet integration is busted so you cannot use VS/VSCode or NuGet CLIs to manage the update of the package and will instead need to do it manually or rely on dependabot.

Reply:
@Evangelink Sorry for the delay, I was out sick. If you understand the problem/limitation, can you reiterate it here in a response to myself? I'll take that info and provide a suggestion.

Reply:
No worries. As I was saying in the previous comment, the limitation is that you cannot use existing tooling to install or update the SDK package (VS nuget package manager won't work, dotnet nuget won't work...). You have to update manually or rely on dependabot.

This problem is not limited to MSTest but is true for any custom MSBuild sdk

Reply:
The dependabot suggestion seems confusing. I don't know how someone would use dependabot, I thought that was just a github thing? Like if I send a copy of my project to a friend through email, what does he do to restore the packages and run the project? How does he actually manually install the required SDK files?

Reply:
Ok let me list scenario:

1/ project is configured to be using MSTest SDK, you open it in VS/VS code or build it in cli then all is good. Restore is working properly, build is working you are all set.

2/ there is a new version of the SDK or some packages referenced by the SDK. You open VS nuget package manager and see only updates for the referenced packages not the SDK. When trying to update those packages they are added to your project but the SDK itself isn't updated and user can only update manually. The story is the same for VS code or cli.

3/ project isn't using MSTest SDK and you want to use it. Then you have to manually update your project files.

For the part about dependabot, yes this is GH only feature but it doesn't understand custom SDK where the version is declared into global.json so if you are using the SDK with version defined there and your code is on GH with dependabot configured, you would get bot PR bumping the version.

Reply:
> yes this is GH only feature but it doesn't understand custom SDK where the version is declared into global.json 

@Evangelink I think you meant **it does understand custom SDK**?

Reply:
@Evangelink @adegeo What do you think of this wording? Does it make it clearer?

```suggestion
The NuGet-provided MSBuild Sdks (including MSTest.Sdk) have limited tooling support when it comes to updating its version, meaning that the usual NuGet update and Visual Studio UI for managing NuGet packages does not work as expected. See this issue for more details: [NuGet#13127](https://github.com/NuGet/Home/issues/13127).
```

Reply:
Ping @adegeo

Reply:
OK this makes more sense now😁 Projects using SDK versions not supplied by the .NET installation, such as updates released only to NuGet, can't be updated using the normal NuGet tooling such as in Visual Studio. You have to manually update the project files to reference a different SDK (or SDK version).

Reply:
>  Projects using SDK versions not supplied by the .NET installation, such as updates released only to NuGet, can't be updated using the normal NuGet tooling such as in Visual Studio. You have to manually update the project files to reference a different SDK (or SDK version).

Correct!

> Where do I go to find information about the latest SDK?

The version of the SDK is aligned with other versions of MSTest so that if you see an update to `MSTest.TestAdapter`, `MSTest.TestFramework` or `MSTest.Analyzers`, you can use the same version number for your SDK. Alternatively, can track our GH repo releases page: https://github.com/microsoft/testfx/releases or just watch NuGet https://www.nuget.org/packages/MSTest.Sdk.

> How do I install the latest SDK?

Manually bump the version in the project or `global.json`. Or if your code is hosted on GH and you are using dependabot, if the version is set in `global.json` it will be updated.